dev-libs/libiodbc/libiodbc-3.52.12.recipe

```diff
@@ -49,14 +52,21 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	runConfigure ./configure --with-iodbc-inidir=$settingsDir/etc
+	runConfigure ./configure --with-iodbc-inidir=$settingsDir/etc \
+		--disable-static
```

Comment:
```suggestion
		--disable-static --disable-libodbc
```

Reply:
OK, could add this suggestion in the browser, but I know I would mess up getting things on par. Checking. :)